conda/recipes/libcudf/meta.yaml

```
@@ -25,21 +25,20 @@ build:
 
 requirements:
   build:
-    - cmake >=3.14.0
+    - cmake=3.17.0
```

This should be changed to `>=3.17.0` before merge:
```suggestion
    - cmake >=3.17.0
```